test/format/ops.c

```diff
@@ -992,6 +993,13 @@ ops(void *arg)
     truncate_op = mmrand(&tinfo->data_rnd, 100, 10 * WT_THOUSAND);
 
     for (intxn = false; !tinfo->quit;) {
+        if (GV(OPS_THROTTLE)) {
+            /* Sleep first to avoid the burst when all threads start. */
+            /* Calculate max delay so that per-table ops/sec is as set. */
+            u_int throttle_delay = 2 * GV(OPS_THROTTLE_SLEEP_US) * GV(RUNS_THREADS) / (ntables > 0 ? ntables : 1);
```

Comment:
Can `ntables` be equal to `0`?

Reply:
Yes, when multitables mode is not enabled (or in older versions). Some examples: [1](https://github.com/wiredtiger/wiredtiger/blob/b462ce6c83863da80c766de69e09a7ad9a72d332/test/format/kv.c#L73) [2](https://github.com/wiredtiger/wiredtiger/blob/b462ce6c83863da80c766de69e09a7ad9a72d332/test/format/format_inline.h#L232).